### PR TITLE
Added the NoStab63 keymap to the keyboard folder

### PR DESCRIPTION
**Pull request by santy81855**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/NoStab63/NoStab63.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include "NoStab63.h"
+
+void matrix_init_kb(void) {
+  matrix_init_user();
+}
```
Comment (drashna) [inline keyboards/handwired/NoStab63/NoStab63.c:5]: This isn't needed.

```suggestion
```

File: keyboards/handwired/NoStab63/config.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (drashna) [inline keyboards/handwired/NoStab63/config.h:7]:
```suggestion
#define PRODUCT_ID      0x1707
```

File: keyboards/handwired/NoStab63/config.h [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Santy
+#define PRODUCT         NoStab63
+#define DESCRIPTION     An Ergonomic Stabless Keyboard
```
Comment (drashna) [inline keyboards/handwired/NoStab63/config.h:11]:
```suggestion
```

File: keyboards/handwired/NoStab63/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+enum layers {
+    _DVORAK,
+	_CAMILA,
+    _NUMBER,
+    _FN1,
+    _ARROW,
+    _LSHIFT,
+    _RSHIFT,
+	_GAMING,
+};
```
Comment (drashna) [inline keyboards/handwired/NoStab63/keymaps/default/keymap.c:12]:
```suggestion
enum layers {
    _DVORAK,
    _CAMILA,
    _NUMBER,
    _FN1,
    _ARROW,
    _LSHIFT,
    _RSHIFT,
    _GAMING,
};
```

File: keyboards/handwired/NoStab63/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+enum unicode_names {
+  SE_AA_HIGH,
+  SE_AE_HIGH,
+  SE_OE_HIGH,
+  SE_AA_LOW,
+  SE_AE_LOW,
+  SE_OE_LOW,
```
Comment (drashna) [inline keyboards/handwired/NoStab63/keymaps/default/keymap.c:20]:
```suggestion
    SE_AA_HIGH,
    SE_AE_HIGH,
    SE_OE_HIGH,
    SE_AA_LOW,
    SE_AE_LOW,
    SE_OE_LOW,
```

File: keyboards/handwired/NoStab63/rules.mk [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+NO_SUSPEND_POWER_DOWN = yes
```
Comment (drashna) [inline keyboards/handwired/NoStab63/rules.mk:21]:
```suggestion
```

File: keyboards/handwired/NoStab63/config.h [hunk 7]
```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+<<<<<<< HEAD
+#define VENDOR_ID       0x0000
+=======
+#define VENDOR_ID       0xFEED
+>>>>>>> 4291fa68390067549e51bb52195ec9cabf6153cb
```
Comment (fauxpark) [inline keyboards/handwired/NoStab63/config.h:10]:
```suggestion
#define VENDOR_ID       0xFEED
```

File: keyboards/handwired/NoStab63/info.json [hunk 8]
```diff
@@ -0,0 +1,118 @@
+{
+    "keyboard_name": "NoStab63",
+    "url": "",
+    "maintainer": "Santy",
+    "width": 18,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_santy": {
+            "key_count": 63,
```
Comment (fauxpark) [inline keyboards/handwired/NoStab63/info.json:9]:
```suggestion
```

File: keyboards/handwired/NoStab63/rules.mk [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = no
+MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+UNICODEMAP_ENABLE = no
+ENCODER_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/handwired/NoStab63/rules.mk:20]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
```

File: keyboards/handwired/NoStab63/rules.mk [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = STM32F303
+
+# Build Options
+#   comment out to disable the options.
```
Comment (fauxpark) [inline keyboards/handwired/NoStab63/rules.mk:5]:
```suggestion
#   change yes to no to disable
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/NoStab63/nostab.h [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+#include "quantum.h"
+
+#define ___ KC_NO
+
+#define LAYOUT_nostab63( \
+	K00, K01, K02, K03, K04, K05,       K06, K07, K08,       K09, K0A, K0B, K0C, K0D, K0E, \
+	K10, K11, K12, K13, K14, K15,       K16, K17, K18,     K19, K1A, K1B, K1C, K1D, K1E, K1F, K1G, \
+	 K20, K21, K22, K23, K24,           K25, K26, K27,        K28, K29, K2A, K2B, K2C, K2D, K2E,  \
+	 K30, K31, K32, K33,                K34, K35, K36,        K37, K38, K39, K3A, K3B,       \
+	                                   K40, K41, K42										  \
```
Comment (noroadsleft) [inline keyboards/handwired/NoStab63/nostab.h:28]: Which matrix position is the encoder click? Seems like it should be `K1F` or `K1G`.

**Comment by noroadsleft** [comment]
This can't be merged with all the other keyboards removed, but since you've deleted the source repository, this PR can't be updated. Closing.

**Pull request closed without merging**